build-tests/api-documenter-test/config/api-extractor.json

```diff
@@ -18,5 +18,14 @@
     "enabled": false
   },
 
-  "testMode": true
+  "testMode": true,
+
+  "messages": {
+    "extractorMessageReporting": {
+      // Purposefully disabled for the `writeonlyProperty` test case.
+      "ae-missing-getter": {
+        "logLevel": "none"
+      }
```

Comment:
I'd prefer to "expect" that there's a single `ae-missing-getter` warning - is that easy to do with how these build tests are set up?

Reply:
No, there isn't an actual test harness here - this is a normal project without any assertions; we just compare its output against Git what's in.

Actually most of the test cases have been moved into the [build-tests/api-extractor-scenarios](https://github.com/microsoft/rushstack/tree/main/build-tests/api-extractor-scenarios) project, which invokes AE separately for each test, and allows each test to have its own `api-extractor-overrides.json` that overrides the default `api-extractor.json` config.  That project doesn't support API doc generation though.

But given that the point of `api-documenter-test` is to invoke API Documenter (versus testing API Extractor edge cases), I'm fine with suppressing `ae-missing-getter` in this project. 👍